lib/spina.rb

```diff
@@ -47,20 +48,20 @@ module Spina
   self.mailer_defaults = ActiveSupport::OrderedOptions.new
   self.thumbnail_image_size = [400, 400]
   self.frontend_parent_controller = "ApplicationController"
-  self.locales = []
+  self.locales = [I18n.default_locale]
```

Comment:
@Bramjetten Will this cause Spina to default to `:en` when the initializer is loaded before any changes to the default locale? I saw there was a recent fix for this in #1022 and wasn't sure if this default would conflict with that.

Reply:
This slipped through, I believe it simply doesn't do anything.

Reply:
Oh my bad, I didn't do that intentionally. I'm not actually sure how that got in there